ui/src/map/map-utils.ts

```diff
@@ -5,7 +5,8 @@ import { BoundingBox, Point } from 'model/geometry';
 
 // offset used for defining a suitable boundingBox around a single location (Point)
 export const MAP_POINT_DEFAULT_BBOX_OFFSET = 178;
-export const MAP_POINT_CLOSEUP_BBOX_OFFSET = 78;
+export const MAP_POINT_NEAR_BBOX_OFFSET = 78;
+export const MAP_POINT_CLOSEUP_BBOX_OFFSET = 38;
```

Comment:
Nimityksiä voi kommentoida. Enpäs tiedä minkälainen nimitysjärjestelmä olisi näille kaikkein selvin.

Reply:
Nämä ovat muuten ihan jees, mutta ainakaan itse en todennäköisesti tietäisi suoraan mitä eroa on NEAR:illa ja CLOSEUP:illa. Kun näitä on ainakin toistaiseksi vaan kolme, niin ehdottaisin default-komparatiivi-superlatiivi-nimeämistä näihin (ts. `...DEFAULT_BBOX_OFFSET`, `...CLOSER_BBOX_OFFSET`, `...CLOSEST_BBOX_OFFSET`)